doc/developers-guide/logging.md

```diff
@@ -5,49 +5,47 @@ To use logger in your module, include
 ```erlang
 -include("mongoose_logger.hrl").
 ```
-
 or
-
 ```erlang
 -include("mongoose.hrl").
 ```
 
 # Logging macros
 
-There are several macroses:
+There are several macros for the most common logging levels:
 
 ```erlang
 DEBUG("event=debug_event info=~1000p", [Arg]),
 INFO_MSG("event=info_event info=~1000p", [Arg]),
-ERROR_MSG("event=error_event reason=~1000p", [Arg]),
 WARNING_MSG("event=warning_event reason=~1000p", [Arg]),
+ERROR_MSG("event=error_event reason=~1000p", [Arg]),
+CRITICAL_MSG("event=critical_event reason=~1000p", [Arg]),
 ```
 
-Use them to log on corresponding log level.
-
-Don't use `ERROR_MSG` for cases that are not errors.
+Use them to log on the corresponding log level.
+Please be mindful of what is logged and which log level is used for it.
 
 # Logging levels
 
-A system operator can choose log level by setting loglevel in `mongooseim.cfg`.
-
-- level 5 - debug
-- level 4 - info
-- level 3 - warning
-- level 2 - error
-- level 1 - critical
-- level 0 - none
+A system operator can choose the global log level by setting `loglevel` in `mongooseim.cfg`.
 
-While there is a macro to log on level 1, we rarely use it.
+- level 8 – all
+- level 7 – debug
+- level 6 – info
+- level 5 - notice
+- level 4 - warning
+- level 3 - error
+- level 2 - critical
+- level 1 - alert
+- level 0 - emergency
 
-If an user sets log level 5, than he would see all messages in logs.
-
-Levels 3 or 2 are usually used for production systems.
+If a user sets log level to `all`, then he would see all messages in logs.
```

Comment:
```suggestion
If a user sets the log level to `all`, then they would see all messages in logs.
```

Reply:
"a user" is singular, so shouldn't we use "he" instead of "they"? Or perhaps we should build the sentence in passive tense ("If the log level is set to..., then all messages would be seen...")?

Reply:
https://en.wikipedia.org/wiki/Singular_they

Reply:
Just use "it". 
Users have no feelings...

Reply:
Set loglevel to `all` to see all the messages in logs.

or something like this ;)  (also shorter)